marine/simple_marine.py

```diff
@@ -0,0 +1,156 @@
+from .marine import Marine
+from . import encap_consts
+from typing import Optional, List, Dict, Tuple
+import sys
+import os.path
+
+marine_instance = {}
```

Comment:
Why is more than one instance needed? I'm pretty sure it actually won't work (only one instance per process)

Reply:
Also what is the use case of loading Marine from multiple paths?

Reply:
Based on recommendations changed so that marine is loaded from default `.so` location (`DL_LIBRARY_PATH`).